From: Maxim Nikulin <m.a.nikulin@gmail.com>
Date: Tue, 28 Sep 2021 18:16:45 -0700
Subject: Fix extra byte on .zip files

Fix extra byte at the tail of recovered zip files if -t all is
used. Use the same signature than the "zip" mode.
---
 state.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/state.c b/state.c
index 5109814..98c11c9 100755
--- a/state.c
+++ b/state.c
@@ -302,7 +302,7 @@ void init_all(f_state *state)
 				 ZIP,
 				 "zip",
 				 "\x50\x4B\x03\x04",
-				 "\x4b\x05\x06\x00",
+				 "\x50\x4b\x05\x06",
 				 4,
 				 4,
 				 100 * MEGABYTE,
